sentry_sdk/scrubber.py

```diff
@@ -1,3 +1,5 @@
+from typing import cast
```

Comment:
How come this doesn't throw an `ImportError` in Python 2.7? I can see that the 2.7 tests are passing but there is no `typing` module in 2.7

Reply:
hm interesting, I was unaware that `typing` did not exist in 2.7, and assumed everything was okay since the tests passed

I will handle the import error, though, so it definitely works in 2.7